pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -67,6 +67,11 @@ func instrumentPluginRequest(ctx context.Context, cfg Cfg, pluginCtx *backend.Pl
 	start := time.Now()
 	timeBeforePluginRequest := log.TimeSinceStart(ctx, start)
 
+	unregisterContextLog := log.RegisterContextualLogProvider(func(ctx context.Context) ([]any, bool) {
+		return []any{"endpoint", endpoint, "dsName", pluginCtx.DataSourceInstanceSettings.Name, "dsUID", pluginCtx.DataSourceInstanceSettings.UID}, true
```

Comment:
Could we also add pluginID?

Reply:
👍  done in 2d533058ca94ca9e164be16105d919fc6e76419f